src/main/java/org/tridiots/ipc/SerializableUtil.java

```diff
@@ -0,0 +1,56 @@
+package org.tridiots.ipc;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class SerializableUtil {
```

Comment:
I'm wondering if we can put all `Util` classes to a single package like `org.tridiots.tdfs.utils` ?
 What's your opinion ?